app-android/src/main/java/com/ustadmobile/port/android/view/CourseTerminologyEditFragment.kt

```diff
@@ -93,4 +104,81 @@ class CourseTerminologyEditFragment: UstadEditFragment<CourseTerminology>(), Cou
             terminologyEntryAdapter?.submitList(value)
             terminologyEntryAdapter?.notifyDataSetChanged()
         }
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+private fun CourseTerminologyEditScreen(
+    uiState: CourseTerminologyEditUiState = CourseTerminologyEditUiState(),
+    onTerminologyTermChanged: (TerminologyEntry?) -> Unit = {},
+    onCtTitleChanged: (String?) -> Unit = {}
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+    )  {
+
+        item{
+            ListItem(
+                text = {
+                    UstadTextEditField(
+                        value = uiState.entity?.ctTitle ?: "",
+                        label = stringResource(id = R.string.name),
```

Comment:
This should be messageIdResource - the messageids that are on common (e.g. terminologyTerm.messageId shared between MUI and Android) have different int values.

Reply:
ok, what is this terminologyTerm.messageId?